app/components/Button/Button.css

```diff
@@ -58,37 +74,83 @@ html[data-theme='dark'] .button {
 }
 
 .dark {
-  background: var(--color-red-3);
-  color: var(--color-white);
-  border-color: var(--color-red-1);
+  background: var(--color-red-2);
+  border: none;
+  color: var(--color-white) !important;
 
   &.fa {
     color: var(--color-white);
   }
 
-  &:active {
-    color: var(--color-white);
+  &:hover:not(:disabled) {
+    background: var(--color-red-1);
+  }
+}
+
+html[data-theme='dark'] .dark {
+  background: var(--color-red-1);
+  color: var(--color-black) !important;
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2) !important;
+  }
+}
+
+.danger {
+  background: none;
+  box-shadow: none;
+  color: var(--color-red-2);
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2);
+    box-shadow: none;
+    border: 1px solid var(--color-red-2);
+    color: var(--color-almost-white-1);
   }
 
   &:disabled {
-    color: rgba(var(--rgb-max), var(--rgb-max), var(--rgb-max), 30%);
+    background: none;
+    color: var(--color-red-2);
+    opacity: 0.5;
   }
 }
 
-.small {
-  padding: 10px 15px;
-  font-size: 0.8rem;
+html[data-theme='dark'] .danger {
+  background: none;
+  box-shadow: none;
+  color: var(--color-red-2);
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2);
+    border: 1px solid var(--color-red-2);
+    color: var(--color-dark-gray-5);
+  }
+
+  &:disabled {
+    color: var(--color-red-2);
+  }
```

Comment:
Most of these are the same as the above `.dark`. Just override the relevant properties if you _have_ to do this. (I would prefer the colors to be correct between dark/light. If some of the darkmode colors can be changed, just do that instead.

Reply:
The `.danger` and `.dark` class do not share any properties. `.danger` is a bit darker than `.dark` 🙃 Should maybe change the name from `.dark` to `.red` or something? Not sure

Reply:
Whoops! I meant `.danger` :sweat_smile:. The point was that the normal `.danger` and `html[data-theme='dark'] .danger` have duplicated a lot of properties.